doc/geany.txt

```diff
@@ -2193,6 +2193,10 @@ Geany can automatically insert a closing bracket and quote characters when
 you open them. For instance, you type a ``(`` and Geany will automatically
 insert ``)``. With the following options, you can define for which
 characters this should work.
+For a more flowing writing (not move the typing hand to arrow keys) Geany
+checks if the character written is the closing quote or bracket that was
+previously inserted. In that case only one quote or bracket of the one written
+and the one previously inserted remains.
```

Comment:
This can be written in a much more concise way. E.g. "To prevent extraneous quotes or brackets Geany detects if you manually insert closing characters.". Then I would place that before the "With the following options, you …" sentence.

Reply:
I disagree, it should be less concise and actually explain what happens, so its known that the behaviour is deliberate, especially as pretty much whatever autoclose behaviour is chosen there are edge cases that are annoying.  It really needs to document it properly.  The commit message is more explicit than the manual.